.github/workflows/daily-nightly-jobs.yml

```diff
@@ -26,20 +29,26 @@ jobs:
         with:
           go-version: 1.18
 
-      - name: teardown minikube and docker
+      - name: teardown minikube, docker and kubectl
+        if: always()
         run: |
-          uptime
           minikube delete
-          docker system prune -a
+          docker system prune --all --force
+          sudo rm -rf  /usr/local/bin/minikube
+          sudo rm -rf /usr/local/bin/kubectl
+
+      - name: remove /usr/bin/yq
+        if: always()
+        run: sudo rm -rf /usr/bin/yq
 
       - name: setup minikube
         run: |
-          # sudo apt-get install build-essential -y
-          # curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-arm64
-          # sudo install minikube-linux-arm64 /usr/local/bin/minikube
-          # sudo rm -f minikube-linux-arm64
-          # curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/arm64/kubectl"
-          # sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+          sudo apt-get install build-essential -y
+          curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-arm64
+          sudo install minikube-linux-arm64 /usr/local/bin/minikube
+          sudo rm -f minikube-linux-arm64
```

Comment:
For these 4 steps I see we are fetching the `minikube-linux-arm64` and then removing it,

SO it does not make sense to have these uncommented

Reply:
we need to clean the machine everytime, that's why I'm remove it